bootstrap_cfn/fab_tasks.py

```diff
@@ -725,7 +731,15 @@ def cfn_create(test=False):
         # So delete the SSL cert that we uploaded
         if 'ssl' in cfn_config.data:
             iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
-        abort('Failed to create stack: {0}'.format(stack))
+        # delete the DNS tag
+        for elb in get_all_elbs():
```

Comment:
I created a PR to fix this chunk of code. basically if elb is empty, txt records won't be deleted. also txt record deletion is missing here. the PR has been merged. you could pull and have a look :p